extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -1299,10 +1333,28 @@ static public function onWikiaRecentChangesBlockHandlerChangeHeaderBlockGroup( $
 			$threadLink = Linker::link( $titleObj, htmlspecialchars( $titleData['articleTitleTxt'] ),
 				[ 'title' => $titleData['articleTitleTxt'] ] );
 
+			$wm->load();
+			$wallMsgTitle = $wm->getMetaTitle();
+
 			$headerTitle = wfMessage( static::getMessagePrefix( $namespace ) . '-thread-group' )
 				->rawParams( $threadLink )
 				->params( $titleData['wallTitleTxt'], $titleData['wallPageName'] )
 				->parse();
+
+			// VOLDEV-3: Bolden pages watched by this user
+			$user = $oChangeList->getUser();
+			// bolding on Watchlist can be removed, on RC, it can not
+			$isWatchlist = $oChangeList->getTitle()->equals( SpecialPage::getTitleFor( 'Watchlist' ) );
+			if ( F::app()->wg->ShowUpdatedMarker && !$user->isAnon()
```

Comment:
This is a huge conditional. Let's break it out into a separate function, like `shouldBoldName` (or something else semantic).